python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -78,6 +81,42 @@ async def dropindex(client: TGlideClient, indexName: TEncodable) -> TOK:
     return cast(TOK, await client.custom_command(args))
 
 
+async def search(
+    client: TGlideClient,
+    indexName: TEncodable,
+    query: TEncodable,
+    options: Optional[FtSeachOptions],
+) -> List[Union[int, Mapping[TEncodable, Mapping[TEncodable, TEncodable]]]]:
+    """
+    Uses the provided query expression to locate keys within an index.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        indexName (TEncodable): The index name for the index to be searched.
```

Comment:
Let's make sure we are aligned with the language used for all these parameters. 
You can just copy the text from commands that are already merged to make your life easier: https://github.com/valkey-io/valkey-glide/pull/2439/files#diff-d522d42e058e72eceba74ae98c9419584280bffc6be7c08f0b4176ebf3c356ed

Reply:
Updated